### ISSUE TITLE
[Assistive tech and device support] Page functionality isn't keyboard accessible. (11.05.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#67441]
#### VFS team name
Virtual Agent

#### VFS product name
Chatbot

#### VFS feature name
Prescription

#### Point of Contact/Reviewers
Ryan Smith - @rsmithadhoc - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Page functionality isn't keyboard accessible.

### Issue Details
Unable to navigate the chatbot via keyboard using Edge + NVDA screen reader. 
    - When tabbing, it opens keyboard controls pop-up and starts reading. Cannot close keyboard controls with the button.
    - "Chat history, press arrow keys to navigate" - pressing arrow keys does not navigate.
    - Cannot press enter to focus a link while the screen reader is active.

### VA.gov Experience Standard
[Category Number 11, Issue Number 05](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.1.1_A

### Platform Recommendation
This may be a result of the chatbot product being used, so I am not sure how much can be done. Ideally, keyboard interactions should work while a screen reader such as NVDA is active.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by Joy-Elizabeth** [comment]
Per staging review, this is a priority for the Chatbot team to address, but not to impact Rx skill launch.
